services/editor/src/pages/security/components/SecurityPage/PoliciesEditor.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { compose, withState, withHandlers } from 'recompose';
+import SaveButton from '../../../../components/common/SaveButton/SaveButton';
+import EditableTable from './EditableTable';
+import './index.css';
+
+const PoliciesEditor = ({
+  savedPolicies,
+  onSave,
+  isSaving,
+  unsavedPolicies,
+  onEdit,
+  onAddPolicy,
+}) => (
+  <div>
+    <div className="action-bar">
+      <SaveButton
+        data-comp="save-button"
+        hasChanges={unsavedPolicies != savedPolicies}
+        isSaving={isSaving}
+        onClick={onSave}
+      />
+      <button className="add-policy-button" data-comp="add-policy" onClick={onAddPolicy}>
+        Add Policy
+      </button>
+    </div>
+    <EditableTable
+      data={unsavedPolicies}
+      onEditTable={onEdit}
+      columns={{
+        user: 'string',
+        group: 'string',
+        contexts: 'json',
+        object: 'string',
+        action: 'string',
+        effect: 'yes-or-no',
```

Comment:
Does it produce allow/deny strings?

Reply:
It is for allow/deny. But in the security page it appears as an on/off switch